dockstore-webservice/src/main/java/io/dockstore/webservice/Helper.java

```diff
@@ -379,75 +389,67 @@ private static Container findGithubCWL(Container c, Map<String, Repository> gitR
      * @param container
      * @param client
      * @param tokenDAO
-     * @param user
      * @return the updated container
      */
-    private static Container findBitbucketCWL(Container container, HttpClient client, TokenDAO tokenDAO, User user) {
+    private static Container findBitbucketCWL(Container container, HttpClient client, Token token) {
         String giturl = container.getGitUrl();
         if (giturl != null && !giturl.isEmpty()) {
-            List<Token> tokens = tokenDAO.findBitbucketByUserId(user.getId());
-            if (!tokens.isEmpty()) {
-                Token token = tokens.get(0);
-
-                Pattern p = Pattern.compile("git\\@bitbucket.org:(\\S+)/(\\S+)\\.git");
-                Matcher m = p.matcher(giturl);
-                LOG.info(giturl);
-                if (!m.find()) {
-                    LOG.info("Namespace and/or repository name could not be found from container's giturl");
-                    return container;
-                    // throw new WebApplicationException(HttpStatus.SC_NOT_FOUND);
-                }
 
-                String url = "https://bitbucket.org/api/1.0/repositories/" + m.group(1) + "/" + m.group(2) + "/branches";
-                Optional<String> asString = ResourceUtilities.asString(url, null, client);
-                LOG.info("RESOURCE CALL: " + url);
-                if (asString.isPresent()) {
-                    String response = asString.get();
+            Pattern p = Pattern.compile("git\\@bitbucket.org:(\\S+)/(\\S+)\\.git");
+            Matcher m = p.matcher(giturl);
+            LOG.info(giturl);
+            if (!m.find()) {
+                LOG.info("Namespace and/or repository name could not be found from container's giturl");
+                return container;
+                // throw new WebApplicationException(HttpStatus.SC_NOT_FOUND);
+            }
+
+            String url = "https://bitbucket.org/api/1.0/repositories/" + m.group(1) + "/" + m.group(2) + "/branches";
```

Comment:
I think you declared the beginning of this as a constant (which was a good idea), use it so that we can easily modify this if say, bitbucket changes the api version number.